app/services/experimentation/runner.rb

```diff
@@ -1,101 +1,12 @@
 module Experimentation
   class Runner
-    PATIENTS_PER_DAY = 10_000
-    BATCH_SIZE = 100
-
-    def self.logger
```

Comment:
Not sure of the context here, but not using the Notifications logger means you can't easily grep all logs w/ a module of `notifications` to follow all logs related to this sub-system. We found this pretty useful when debugging a/b testing initially.

Reply:
I suspect logging will go deeper in the code, and we might see this resurface elsewhere. Anyhow, I do think we should keep this in mind when we solve for observability for this round of enhancements.

Reply:
This was helpful indeed 👍🏼, I will probably add a similar logger to the new runner in a follow up PR.